### ISSUE TITLE
IE11 Support and Accessibility surrounding <main>

### ISSUE THREAD

**Issue by votemike** [vercel/next.js#25527]
### What version of Next.js are you using?

10.0.0

### What version of Node.js are you using?

12.0.0

### What browser are you using?

IE11

### What operating system are you using?

Windows

### How are you deploying your application?

next start

### Describe the Bug

For accessibility reasons, all websites should have an element with role="main". NextJS has a `<main>` element which browsers that support `<main>` automatically give the role of main.
IE11 however doesn't support `<main>` and therefore there is no element with a role of main.
According to https://nextjs.org/docs/basic-features/supported-browsers-features, NextJS supports IE11.
Either `<main>` should be updated to `<main role="main">` or NextJS should remove statement stating that it supports IE11 (Internet Explorer is being sunsetted soon anyway).

### Expected Behavior

I expect any tool that supports IE11 to have a landmark of role "main".

### To Reproduce

Tools such as bbc-a11y point out this problem. See https://github.com/bbc/bbc-a11y/pull/328#issuecomment-772701224 for more information.

**Comment by votemike** [comment]
As of NextJS 13, IE11 is no longer supported: https://nextjs.org/docs/basic-features/supported-browsers-features
